glean-core/rlb/src/lib.rs

```diff
@@ -760,5 +771,17 @@ pub fn set_source_tags(tags: Vec<String>) -> bool {
     }
 }
 
+/// Load the dynamic library and dispatch all metric recordings to that library using the Glean
+/// FFI.
```

Comment:
We're gonna need a more distinct name than "FFI" for this since we have (at last count) 2 (specific metrics FFI for underpinning non-RLB LBs, MLA FFI for FOG) plus this one... "dynFFI"? I dunno, I'm not a marketer.